### PR TITLE
Initial files

### PR DESCRIPTION
**Pull request by core23**
Just the content of https://github.com/core23/ClassificationMediaBundle with the correct namespace.

### CODE REVIEW

**Comment by greg0ire** [comment]
Most of these files are supposed to be delivered by dev-kit, right? Why not setup dev-kit first?

**Comment by core23** [comment]
Still waiting for @rande to do this: https://github.com/sonata-project/SonataClassificationMediaBundle/issues/2

**Comment by OskarStark** [comment]
@core23 any news here? #2 is merged

and shouldn't we create a release afterwards?

**Comment by core23** [comment]
First of all we should merge this stuff and test it

**Comment by greg0ire** [comment]
> Most of these files are supposed to be delivered by dev-kit, right? Why not setup dev-kit first?

What about this?

**Comment by core23** [comment]
There is PR for this: https://github.com/sonata-project/dev-kit/pull/193

**Comment by greg0ire** [comment]
Shouldn't we wait for a dev-kit deployment before reviewing this?

**Comment by core23** [comment]
You'r right. Thought the deployment did run, because the dev-kit PR was merged.

**Comment by greg0ire** [comment]
I'm on my phone can't find said PR or any other file than the composer manifest. What's the deal?

**Comment by core23** [comment]
@Soullivaneuh Can you have a look at the cronjob? We don't get any dev-kit updates for this repo :(

**Review by OskarStark — APPROVED** [review]

**Comment by core23** [comment]
@sonata-project/contributors Please review and merge. We need this bundle for the next major versions of the media and classification bundle

**Comment by OskarStark** [comment]
LGTM, but is it fine, that travis failed?

```
PHP Parse error:  syntax error, unexpected 'rh' (T_STRING) in /home/travis/build/sonata-project/SonataClassificationMediaBundle/tests/autoload.php.dist on line 39
PHP Stack trace:
PHP   1. {main}() /home/travis/.phpenv/versions/5.6.5/bin/phpunit:0
PHP   2. PHPUnit_TextUI_Command::main() /home/travis/.phpenv/versions/5.6.5/bin/phpunit:722
PHP   3. PHPUnit_TextUI_Command->run() phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/TextUI/Command.php:104
PHP   4. PHPUnit_TextUI_Command->handleArguments() phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/TextUI/Command.php:114
PHP   5. PHPUnit_TextUI_Command->handleBootstrap() phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/TextUI/Command.php:622
PHP   6. PHPUnit_Util_Fileloader::checkAndLoad() phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/TextUI/Command.php:792
PHP   7. PHPUnit_Util_Fileloader::load() phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/Util/Fileloader.php:42
PHP   8. include_once() phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/Util/Fileloader.php:58
Parse error: syntax error, unexpected 'rh' (T_STRING) in /home/travis/build/sonata-project/SonataClassificationMediaBundle/tests/autoload.php.dist on line 39
```

**Review by OskarStark — APPROVED** [review]

File: tests/autoload.php.dist [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    die('Unable to find autoload.php file, please use composer to load dependencies:
+
+wget http://getcomposer.org/composer.phar
+php composer.phar install
+
+Visit http://getcomposer.org/ for more information.
+
+');
+}
+g rh
```
Comment (jordisala1991) [inline tests/autoload.php.dist:39]: this makes travis fail

**Review by jordisala1991 — CHANGES_REQUESTED** [review]

File: tests/autoload.php.dist (hunk 1 above)
Comment (OskarStark) [reply tests/autoload.php.dist:39]: strange, but this is not part of the file itself:
https://github.com/core23/ClassificationMediaBundle/blob/master/Tests/tests/autoload.php.dist

Comment (jordisala1991) [reply tests/autoload.php.dist:39]: let's merge and fix afterwards?

Comment (OskarStark) [reply tests/autoload.php.dist:39]: yes 👍 
will you do it?

**Comment by OskarStark** [comment]
Thank you @core23 👍

**Pull request merged**